테라폼 community 가이드 작성(25년 2월) (#20)
nhn-cloud/terraform-provider-nhncloud: docs/resources/nhncloud_lb_loadbalancer_v2.md

* `vip_address` - (Optional) The IP address of the load balancer.
* `security_group_ids` - (Optional) The list of security group IDs to be applied for the load balancer.<br>**Security groups must be specified by ID, not by name**.
* `admin_state_up` - (Optional) Administrator control status.

> * `loadbalancer_type` - (Optional) It is a load balancer type, which can be used as `shared` or`dedicated`

Reviewer: ```suggestion
* `loadbalancer_type` - (Optional) The load balancer type that can be used as `shared` or `dedicated` and set as `shared` if omitted.
```

> and set as `shared` if omitted.

Reviewer: ```suggestion
```
